src/site/stages/build/process-cms-exports/schemas/common/transformed-address.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  $id: 'Address',
+  type: ['object', 'null'],
+  // None of these are required
```

Comment:
Should we require some of them?

Reply:
Maybe. I don't know which are consistently required, though. So far, only `addressLine1` seems to be, but... :man_shrugging: